pkgs/applications/misc/koreader/default.nix

```diff
@@ -46,7 +46,7 @@ in stdenv.mkDerivation rec {
     description =
       "An ebook reader application supporting PDF, DjVu, EPUB, FB2 and many more formats, running on Cervantes, Kindle, Kobo, PocketBook and Android devices";
     platforms = intersectLists platforms.x86_64 platforms.linux;
-    license = licenses.agpl3;
+    license = licenses.agpl3Only;
```

Comment:
The source for the Debian package is mentioning [additional licenses](https://github.com/koreader/koreader/blob/master/platform/debian/copyright).

Reply:
Looking at this file it seems only fonts and external programs use other licenses, KOreader itself is agpl3Only.

Reply:
I checked the GitHub repository and does not allow later versions so the `agpl3Only` seems to be the correct one to use.